pkg/settlement/swap/transaction/transaction.go

```diff
@@ -154,6 +154,13 @@ func (t *transactionService) WaitForReceipt(ctx context.Context, txHash common.H
 	}
 }
 
+func IncreaseGasPrice(gasPrice *big.Int) *big.Int {
+	// increase 40%
+	gasPrice.Div(gasPrice, big.NewInt(5))
+	gasPrice.Mul(gasPrice, big.NewInt(7))
```

Comment:
why not considering switching the order of these two lines to avoid cases such as 2 / 5 * 7 = 0 and instead make it 2 * 7 / 5?